app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
```

Comment:
should this stay in?

Reply:
@nickytonline: thoughts on this change from your commit?